src/R/Components/Impl/Plots/Implementation/View/RPlotHistoryControl.xaml.cs

```diff
@@ -33,6 +37,11 @@ public partial class RPlotHistoryControl : UserControl {
             if (e.Key == Key.Enter) {
                 var entry = (IRPlotHistoryEntryViewModel)(((ListViewItem)sender).Content);
                 ActivatePlot(entry);
+            } else if (e.Key == Key.Apps || (e.SystemKey == Key.F10 && (Keyboard.Modifiers & ModifierKeys.Shift) == ModifierKeys.Shift)) {
+                ListViewItem item = (ListViewItem)sender;
+                var point = item.TranslatePoint(new Point(1, 1), this);
```

Comment:
The event is attached to ListViewItem directly, it won't fire if it's not a ListViewItem or if null.
```
            <ListView.ItemContainerStyle>
                <Style TargetType="ListViewItem">
                    <EventSetter Event="MouseDoubleClick" Handler="item_MouseDoubleClick"/>
                    <EventSetter Event="KeyUp" Handler="item_KeyUp"/>
                </Style>
            </ListView.ItemContainerStyle>
```